doc/DeviceAssociationInLocalNetwork.md

```diff
@@ -0,0 +1,188 @@
+# Device Association in Local Network
```

Comment:
General notes for the docs I fixed in my previous commits:

- Titles / Headings should be in sentence case, according to the TW authorities.  I was very resistant to this for a long time, but now just do it automatically.
- Please wrap the paragraphs at 65 characters.  It doesn't affect how they render, but makes them more readable / editable in a text editor.